client-gateway/bot/passenger.py

```
@@ -1,40 +1,35 @@
-<<<<<<< HEAD
 import logging
 import re
+import time
+from datetime import datetime
 from telegram import Update, InlineKeyboardMarkup, InlineKeyboardButton, ReplyKeyboardRemove
 from telegram.ext import MessageHandler, CallbackQueryHandler, ConversationHandler, ContextTypes, filters, CommandHandler
+from telegram.helpers import escape_markdown
```

Flake8 (7.3.0)

[error] 'telegram.helpers.escape_markdown' imported but unused

(F401)